accounts/serializers.py

```diff
@@ -41,14 +41,9 @@ def validate(self, data):
         
         data['user'] = user
         return data
-        
-        
-    
 
-        
 
-        
-        
-    
-    
-    
+class UserSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = User
+        fields = ['username', 'email']
```

Comment:
Here you have created a UserSerializer class in order to serialize data from the database to json file with class Meta that contains metadata.